locale/zh-cn/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: index.hbs
 labels:
-  banner: 重要的安全版本，请立即更新！
+  banner: 崩溃和幽灵(漏洞) - 对Node.js的影响
```

Comment:
It's better to use English original for those two security issues, since people are more familiar with its English name. Good to me for current translated content, and keep it going.